download.sh

```diff
@@ -38,6 +37,17 @@ Exit codes:
 set -euo pipefail
 # set -x
 
+build_user_agent() {
+    # While the dump websites are not part of the API, it's still polite to identify yourself.
+    # See https://meta.wikimedia.org/wiki/User-Agent_policy
+    subcommand=$1
+    name="OrganicMapsWikiparserDownloaderBot"
```

Comment:
Can spaces be added here?

Reply:
I haven't seen an example with spaces in the name. All of the browser user agents use CamelCase instead of spaces.